cogboard-webapp/src/components/Board/index.js

```diff
@@ -3,14 +3,16 @@ import { useSelector, useDispatch } from 'react-redux';
 
 import { setCurrentBoard } from '../../actions/actionCreators';
 
-import WidgetList from '../WidgetList';
 import { StyledContainer, StyledNotFound, StyledNoBoards } from './styled';
+import boardTypes from '../boards';
 
 const Board = ({ boardId, className }) => {
   const currentBoard = useSelector(({ boards }) => boards.boardsById[boardId]);
   const allBoardsLength = useSelector(({ boards }) => boards.allBoards).length;
-  const { columns, widgets } = currentBoard || {};
   const dispatch = useDispatch();
+  const BoardType = currentBoard.type
+    ? boardTypes[currentBoard.type].component
+    : '';
```

Comment:
`WidgetBoard` as a default here?

Reply:
Yes, good suggestion. This is the part that I have mentioned yesterday and will be working on today - to provide some backward compatibility.

Reply:
Done